### PR TITLE
added solution

### PR DESCRIPTION
**Pull request by floriangeyer21**

### CODE REVIEW

File: src/main/webapp/WEB-INF/views/injectData.jsp [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+            text-align: center;
+        }
+        .font1 {
+            font-size: 120%;
+        }
+    </style>
+</head>
+<body>
+<link href="${pageContext.request.contextPath}/resources/style4.css" rel="stylesheet" type="text/css">
+<h1 class="font">Your mock data waa added to DB</h1>
```
Comment (vremere-a) [inline src/main/webapp/WEB-INF/views/injectData.jsp]: was

**Review by vremere-a — APPROVED** [review]

File: src/main/java/com/internet/shop/controllers/cart/DeleteShoppingCartController.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+import com.internet.shop.service.ShoppingCartService;
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/shoppingCart/products/delete")
+public class DeleteShoppingCartController extends HttpServlet {
```
Comment (RomanDubovskyi) [inline src/main/java/com/internet/shop/controllers/cart/DeleteShoppingCartController.java]:
```suggestion
public class DeleteProductFromShoppingCartController extends HttpServlet {
```

File: src/main/java/com/internet/shop/controllers/user/DeleteUsersController.java [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    private static final Injector injector = Injector.getInstance("com.internet.shop");
+    private final UserService userService =
+            (UserService) injector.getInstance(UserService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        String userId = req.getParameter("id");
+        Long id = Long.valueOf(userId);
+        userService.delete(id);
```
Comment (RomanDubovskyi) [inline src/main/java/com/internet/shop/controllers/user/DeleteUsersController.java:28]: let's delete shopping cart here as well

File: src/main/java/com/internet/shop/controllers/user/UsersController.java [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+import com.internet.shop.model.User;
+import com.internet.shop.service.UserService;
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class UsersController extends HttpServlet {
```
Comment (RomanDubovskyi) [inline src/main/java/com/internet/shop/controllers/user/UsersController.java]:
```suggestion
public class GetAllUsersController extends HttpServlet {
```

File: src/main/java/com/internet/shop/controllers/user/DeleteUsersController.java [hunk 5]
```diff
@@ -14,12 +15,15 @@
...
             (UserService) injector.getInstance(UserService.class);
+    private final ShoppingCartService shoppingCartService =
+            (ShoppingCartService) injector.getInstance(ShoppingCartService.class);
 
     @Override
     protected void doGet(HttpServletRequest req, HttpServletResponse resp)
             throws ServletException, IOException {
         String userId = req.getParameter("id");
         Long id = Long.valueOf(userId);
+        shoppingCartService.delete(id);
```
Comment (RomanDubovskyi) [inline src/main/java/com/internet/shop/controllers/user/DeleteUsersController.java]: you delete cart by userId, find cartId first and then pass it to delete method

**Review by RomanDubovskyi — APPROVED** [review]

**Pull request merged**